test/mocha/20-issue.js

```diff
@@ -24,6 +24,16 @@ describe('issue', () => {
       },
       tags: ['general']
     },
+    'eddsa-jcs-2022': {
+      suiteName: 'eddsa-jcs-2022',
+      algorithm: 'Ed25519',
+      issueOptions: {},
+      statusOptions: {
+        suiteName: 'eddsa-jcs-2022',
+        algorithm: 'Ed25519'
+      },
+      tags: ['general']
```

Comment:
I think we only need one suite to run through the general tests, no need to duplicate them -- or is there a good reason to add this one in particular?

```suggestion
      tags: []
```